cypress/integration/userCanReadASpecificArticle.feature.js

```diff
@@ -37,7 +56,7 @@ describe("User can", () => {
     });
   });
 
-  context("not see an article if improper params are provided", () => {
+  describe("not see an article if improper params are provided", () => {
```

Comment:
why did you change this from `context` to `describe`?

Reply:
We have describe in other tests, so it was just for consistency!